api_app/serializers.py

```diff
@@ -179,10 +194,48 @@ def validate(self, attrs: dict) -> dict:
         file_obj.seek(0)
         file_buffer = file_obj.read()
         attrs["md5"] = calculate_md5(file_buffer)
+        self.filter_analyzers_and_connectors(attrs)
         logger.debug(f"after attrs: {attrs}")
         return attrs
 
 
+class MultipleObservableAnalysisSerializer(rfs.ListSerializer):
+    """
+    ``Job`` model's serializer for Multiple Observable Analysis.
+    Used for ``create()``.
+    """
+
+    def update(self, instance, validated_data):
+        raise NotImplementedError("This serializer does not support update().")
+
+    child = None
+    many = True
+    observables = rfs.ListField(required=True)
+
+    def to_internal_value(self, data):
+        ret = []
+        errors = []
+
+        common_data = dict(data)
```

Comment:
is this cast really needed? which kind of objects you get here?

Reply:
Removed